pandas/core/window.py

```diff
@@ -899,11 +899,45 @@ def skew(self, **kwargs):
         return self._apply('roll_skew', 'skew',
                            check_minp=_require_min_periods(3), **kwargs)
 
-    _shared_docs['kurt'] = """Unbiased %(name)s kurtosis"""
+    _shared_docs['kurt'] = dedent("""Calculate unbiased %(name)s kurtosis.
 
-    def kurt(self, **kwargs):
+This function uses Fisher's definition of kurtosis (kurtosis of normal
```

Comment:
I think by using `dedent`, you can actually keep this indented (so it doesn't look that 'off' inside the class)

(I am not even sure the `dedent` is needed, as there are many places where we don't use it, but that is something else)

Reply:
Yep mistake on initial push. Subsequently indented in the next push